### PR TITLE
Collapse the short description module by default

### LINKED ISSUE #33537: Product creation experience: no clear distinction between long and short description
**Description:** we don't offer any context about the difference between the long and short description fields. This can lead the user to use one of the fields with the wrong expectations, which can lead to errors and slow down the product creation process.

**Acceptance criteria:**
- [x] Improve the empty state of each field with copy explaining the differences or add a tooltip
- [x] Collapse the short description module by default
- [ ] Change the short description name to "Summary". This change can be risky as it will require updating documentation and screenshots. It should be treated as a nice to have

### PR DESCRIPTION
**Pull request by mdperez86**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #33537.

### How to test the changes in this Pull Request:

1. Visit the `Add new` product page
2. When hover the ? icon next to `Add media` button a tooltip should show like below:
image
3. When hover the ? icon next to panel with title `Product short description` a tooltip should show like below: 
image
4. Every time the page is visited or refreshed the `Product short description` panel should remain collapsed by default. 
image

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce/client/legacy/css/admin.scss [hunk 1]
```diff
@@ -7955,3 +7749,24 @@ table.bar_chart {
 		}
 	}
 }
+
+#wp-content-media-buttons {
```
Comment (mdperez86) [inline plugins/woocommerce/client/legacy/css/admin.scss:7753]: Changes in this file starts from this line. Other changes were caused by the linter.

File: plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js [hunk 2]
```diff
@@ -759,5 +904,38 @@ jQuery( function( $ ) {
 		$( '#tiptip_arrow' ).removeAttr( 'style' );
 
 		return false;
-	});
-});
+	} );
+
+	// Add a descriptive tooltip to the product description editor
```
Comment (mdperez86) [inline plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js:897]: Changes in this file starts from this line. Other changes were caused by the linter.

**Review by octaedro — APPROVED** [review]
Great job @mdperez86! This is testing well here and the code LGTM :shipit: 
Thank you for fixing the `Product data` selector :-)

**Comment by mold-man1** [comment]
Hi everyone, please reconsider collapsing the short description by default. To view the reasons why our discussion group believes this is not a good idea, please see our comments here- https://wordpress.org/support/topic/how-to-expand-short-description-by-default/

**Comment by loulemos** [comment]
Good morning, I'd like for this to be seriously considered, I have 3 clients complaining about it as it ruins their workflow. At least give us an option in Woocommerce to enable/disable by default. Since the update there have been at least 5 threads so far posted in the Woocommerce support forum and I don't think they're going to stop anytime soon. To be frank, who had this bloody ideia?

**Comment by michaeljnissen** [comment]
Thank you for the quick fix @louwie17! Just in case another debate breaks about this topic in the future, here are my two cents: I also had clients complain about this. I believe the short description is elementary especially from a customer and conversion point of view. I have several smaller customers (and this is probably a significant niche for WC) who do not use the regular description at all, but instead simply use 2-3 sentences as short description. Frankly, I would rather want to see the short description edit field be placed before the regular description. This would i) highlight the importance of the field and ii) be more concise with the position where most themes display the short description frontend-side (usually before the regular description). I just see that @pmcpinto mentioned the same suggestion/issue in the original pull request originally leading to this change.

**Comment by rinkrote** [comment]
Hi First time github user/post here! 👋 I'd like to ask that the auto-collapsed short description be reconsidered. (edit: I now see the other github pull request that reverts! I'm happy to see that) I am a web designer and help small business owners build ecom on Woo. I use the short description on EVERY product listing AND I educate my clients on the best practice of keeping it short or highlight a few key features using a bulleted list. By having this section auto-collapsed, it adds an extra click to every product. I never knew until yesterday how inconvenient one extra click could be. Please reconsider this decision. I do love the tooltip with best practice suggestion. Maybe having the SD box be a bit smaller or having placeholder text that shows an example of highlighting 3 short bullet points. Or have some education included in a startup wizard/walkthrough? Thank you for all your hard work in developing Woo, I use it almost daily. Thanks for your time and consideration

**Comment by pmcpinto** [comment]
Thanks for your feedback. We'll revert this change in WooCommerce 7.2 (December)

**Comment by GlazerGallery** [comment]
michaeljnissen wrote: > Frankly, I would rather want to see the short description edit field be placed before the regular description. 

I completely agree. That's the position in which it appears on the actual product page. I have never understood why the layout of the Edit Page is reversed. It makes no intuitive sense. It also is an annoying quirk since the business owner I work for writes up the info for the product pages in Word documents that we copy-paste into those fields -- a list of the main features and a short intro paragraph into the Short Description, and then the details, if there are any (and sometimes there aren't) into the long description part. I've gotten used to scrolling down to paste the Short Description info in when I open an Edit Page, and then scrolling back up, and have never liked that I had to do that. Moreover, when opening the Edit Page if there _is_ no long description, you're greeted with a blank box on opening it. So as long as we're questioning collapsing Short Description, I think putting it at the top is a change that's long overdue for a smoother workflow. And it would also help new users if part of the goal is making this more intuitive to use.

**Comment by mold-man1** [comment]
I used to think the same thing. But I recently discovered that you can drag the short description field to the top.

**Comment by mold-man1** [comment]
Oh, and by the way, my short description field is now much taller and not collapsed by default. I modified the code in certain files. If you do not want to wait for the next WordPress update, I explained how I did that in the WordPress forum at - https://wordpress.org/support/topic/how-to-expand-short-description-by-default/. See my comments there (user name dave8441). I hope this helps.

**Comment by GlazerGallery** [comment]
Hi mold-man 1 -- you wrote > I used to think the same thing. But I recently discovered that you can drag the short description field to the top.

That does not work for me. I've tried before and I just tried again. It will not move. The closest I can get is right under the long description.

**Comment by mold-man1** [comment]
Hi @GlazerGallery ,
This might work... at the top of the edit product page, set the screen options to two columns if it isn't already. Then move the short description to the side column by clicking on the "v". Then try dragging it up. I think that is how I did it if I recall.

**Comment by GlazerGallery** [comment]
Hi @mold-man1 -- thanks for the idea, but that doesn't work for me. I do have the two-column layout enabled already, but the way the Edit Page in WooCommerce is set up, the left column takes up about 3/4 of the width, and having the Short Description squished into that narrow column on the right doesn't work for me. I just want it directly above the long description.

**Comment by mold-man1** [comment]
I meant to say, after you put it in the narrow column, do not leave it there, but then try to drag it from there to directly above the long description. After you do that, the width of the short description will automatically widen to the same width as the long description.

**Comment by GlazerGallery** [comment]
Hi @mold-man1 -- No, that does not work for me. I can put it in the narrow column, but cannot move it back to the wide column above the long description. The closest I can get is right under the long description.

**Comment by mold-man1** [comment]
Sorry that did not work for you. You may want to post the issue at https://wordpress.org/support/plugin/woocommerce/ and perhaps the plugin authors can help.

**Comment by myedgy** [comment]
Please revert this! Super annoying to reopen the 'short description field' each time, its just an extra click. And it's small thay i always scroll over it to find it.. It's just an extra step. 

If you don't want to see this field. Drag it to the bottom of your dashboard panel.

Please revert this, thank you.

**Comment by pmcpinto** [comment]
This change will be reverted in WooCommerce 7.2: #35213

**Comment by mold-man1** [comment]
DocuSign

Your document is ready 
 VIEW COMPLETED DOCUMENT 
 
 
Confidential information intended only for the use of the individual or entity named above. If you have received this as error, please notify the sender immediately and delete from your email. Any unauthorized disclosure, copying, distribution, or use of the information contained in this fax is strictly prohibited. 































 

 

 

 

 

 

 

 

 

 

 

 

 

 

 

 

 

 




Mario Turcios O. 















On Friday, March 28, 2025, 2:48 PM, ahceb_tac . ***@***.***> wrote:





Dear Sir Jerome/Sir Boy,
 
Requesting approval to allow PMR and PSR to handcarry 10 boxes of 161 to Vicente Sotto Memorial Medical Center.
SO was just approved today, 1:30 PM, and all trucks are already out for delivery. VSMMC only need stocks that will last until Monday before the delivery of the remaining stocks.
 
Reference SO: 767272 
 
We would like to ask approval because the hospital badly needed the stocks. 
 
Thank you,
 
 


Narciso, Ivy P.
Acting Area Head - Cebu/Tacloban
Euro-Med Laboratories Phil. Inc.
















On Friday, March 28, 2025 at 02:36:50 PM GMT+8, ceb_03 . ***@***.***> wrote:
 
 

Good Afternoon Maam Ivy, 
 
I would like to request handcarry 10 boxes of Sterile water 1L twist-off to deliver at Vicente Sotto Mem Medical Center because our forwarder already departed from depot lately to deliver other locations and there's  no exact time when they can probably return for loading  The account is urgently need our 161 and hoping for us to deliver even partial of 10 boxes. These 10  boxes will be carried by our sales rep Ronald Jr Simbajon and Ryelle Kim Limpag just to fullfil the urgent need of Mr Froi Head nurse in Respiratory unit. 

Thank you for your kind consideration! 
 

Jullivie C. Hipulan
Team Leader Cebu South/Bohol
Euro-Med Laboratories Phil., Inc.

**Comment by mold-man1** [comment]
Signature Request 
            forContract Settlement Offer Letter
            Open agreement
                        
            
              
              
                 
              
                
                  Attached is the final agreement for your 
                  reference. You can also open it 
                  online to review its activity 
                  history.
  
    
      
      
        
        
          
            
            
            
              
              
                
              
                 
            Need 
            your own documents signed? Adobe Acrobat Sign can help save you 
            time. Learn more.
            To 
            ensure that you continue receiving our emails, please add 
            ***@***.*** to your address book or safe 
list.
            © 
            2024 Adobe. All rights reserved.

**Comment by mold-man1** [comment]
***@***.***,
You have received a document to review and sign




REVIEW DOCUMENT
All document is pending Please DocuSign: Declaration and Final Release. pdf

Please Review & Complete with Docusign:Thanks


If you are unable to Review document file link, please move message to Inbox folder.



Powered by 

Do Not Share This EmailThis email contains a secure link to Docusign. Please do not share this email, link, or access code with others.About Docusign
Sign documents electronically in just minutes. It's safe, secure, and legally binding. Whether you're in an office, at home, on-the-go -- or even across the globe -- Docusign provides a professional trusted solution for Digital Transaction Management™.Questions about the Document?
If you need to modify the document or have questions about the details in the document, please reach out to the sender by emailing them directly.This message was sent to you by Cali Suderman who is using the Docusign Electronic Signature Service. If you would rather not receive email from this sender you may contact the sender with your request.

**Pull request merged**